workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,127 @@
+
+#### Inner Value Aggregation
+
+The Inner Value Aggregation is a nested aggregation technique that allows you to aggregate your initial query. The below example is grouping the `k8s.node.memory.usage` metric by summing the `k8s.cluster.name` attributes together and aggregating __Any__ `k8s.node.memory.usage`:
+
+![](https://lh7-us.googleusercontent.com/fo5Img7dsK14nN-w6atI390lqE-Lrg9oHS6BlqlgLwJKjNjpke85oDeEVYfhbl3CGj3hPiZ6mfnK5XXGPZGlxbfzY5wjd31myEhAR4gAJnQgB5Eii1Ex7CvkQgufEmUNFVcKZiLkUBFt35WBIGRtBHc)
+
+#### Filter
+
+The filter feature is used to remove unwanted metrics within your query. The below example is filtering out any `host.id` containing `ip-172-31-13-218.us-east-2.compute.internal`:
+
+![](https://lh7-us.googleusercontent.com/_50CgdVxjgsqeqS6U3S8Rud9llopVXSi97d2WXCo4shRYJp79HjUs8gGTq_ZLihDDlP4wpi3IqCvaT6z4NpnhXAQWzhNWbe2uUS89Gw4Nd8ww_RX95Ce5p4LsR_SBO3-0LSp3sAht3heL01QDKgCAQQ)
+
+#### Advanced
+
+The Advanced section allows you to specify the interval at which your data is ingested and displayed at.
+
+<Note> The default Data Aggregation Interval is assumed to be every 15 seconds unless specified. </Note>
+
+#### Alias
+
+The Alias feature allows you to overwrite the metric name displayed in your widget for increased readability:
+
+![](https://lh7-us.googleusercontent.com/8gAYYTNk5B9dH_6a6S6DLq-zItBxQU_pv0GBuM0PnOonqY9kbxMiESmKRNa_vG6R7vDrhl5ELX_nRGloIlsHm5VDBM1xRuUuO7Az8wZk3orJnG6yrvTGeA8Nul_XDGugYy4_NnEavJISDxFtMdcMVyw)
+
+### Step 3: Set Display Preferences
+
+The Display Preferences section is the default time range that is displayed when you open the dashboard.
+
+<Note> Although this is the default time range, drill into other time ranges with the Date Range filter. </Note>
+
+### Step 4: Enter Widget Title
+
+The Widget Title is the title that will be displayed at the top of your widget on the dashboard.
+
+# Clone Dashboards & Widgets
+
+The clone feature allows you to experiment with new or slightly modified dashboard designs without disrupting your existing setup. Save time by cloning widgets instead of recreating them from scratch. 
+
+To clone a widget, click on the elipses in the top right corner of the widget and select `clone`:
+
+![](https://lh7-us.googleusercontent.com/Y5euVq5Qju0bzuDRpwUpM1_8TFsWda4wL4WlRvnyYYYPP0DoLbrIfIE7_uNeRiqz6le4qI1EjkO000QEeBLNduUNDHrZY0oIs2C1Oi4bSYzhG2SvEvJDSobjr6U8IC3-kzwzbRj4Yfm7xsxJ0Q4wQjo)
+
+To clone a dashboard, click on the elipses located in the dashboards table under `Actions` and select `clone`:
+
+![](https://lh7-us.googleusercontent.com/fMghA-ye5DNOrPlybyWCJhWBOGlPHJInIhX9uwiUnHJR42_ZMSeOeoG6w75rh6AQCW3rbG-sIKStnHSoW4zjISQwwy6GqzPxAd5z6Dv-8Fa86iUpsOvBrjCM3TU-Xuh_Bs1InLwtQqvcL_j06m_u80s)
+
+# Filter Dashboard Data
+
+### Live & History View
+
+View your static dashboard data in __History__ mode and view your data in real time in __Live__ mode.
+
+### Date Range
+
+The Date Range picker allows you to select specific period of time to view your dashboard data.
+
+### Metrics
```

Comment:
Recommend calling this section "Filters" and talk about how to apply them, and where the filters come from, e.g. "The list of available filters is automatically built by the metadata pulled from all metrics being queried on the dashboard."